pkg/build/build.go

```diff
@@ -67,3 +72,30 @@ func sourceVersion(kubeRoot string) (string, error) {
 	}
 	return "", fmt.Errorf("could not find kubernetes version in output: %q", strings.Join(output, "\n"))
 }
+
+var (
+	CommonTestBinaries = []string{
```

Comment:
Any reason this needs to be a `var`?
```suggestion
const (
	CommonTestBinaries = []string{
```

Reply:
go doesn't support array constants? 
https://golang.org/ref/spec#Constants